app/models/dish.rb

```diff
@@ -1,16 +1,14 @@
 class Dish < ApplicationRecord
+  validates :name, presence: true
+  validates :price, presence: true
 	has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
-  validates_attachment_content_type :image, content_type: /\Aimage\/.*\z/	
-
+  validates_attachment_content_type :image, content_type: /\Aimage\/.*\z/
 
   def dish_image
-  	if image.present?
-  		 image.url(:medium)
-		else
-			 'dish.svg'
-  	end
+    if image.present?
+      image.url(:medium)
+    else
+      'dish.svg'
+    end
```

Comment:
you can use new ruby feature here:
```ruby
image&.url(:medium) || 'dish.svg'
```
`xx&.something` will firstly check if xx is present, then if it is it will try to use method `something` on contents of `xx`, otherwise it will just return nil, this behaviour is similar to `xx.try(:something)`. Check this out: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/